### DISCUSSION TITLE
Contributions on soroban-example-dapp to make it more Solidity-dev friendly :)

### DISCUSSION THREAD

**Discussion by esteblock** [stellar/sorobanathon#71] [General]
Hello!
I have been working on some contributions on the soroban-example-dapp repo (https://github.com/stellar/soroban-example-dapp/)
I made two pull-requests: https://github.com/stellar/soroban-example-dapp/pull/62 https://github.com/stellar/soroban-example-dapp/pull/64

The most important is [this one](https://github.com/stellar/soroban-example-dapp/pull/64), where I focused my work on the Soroban Connector (now only Freighter) interaction with ReactJS changed some function names and behaivour in order to make it more [web3-react](https://github.com/Uniswap/web3-react) similar.

As web3-react is very used among solidity-devs, this makes the project more solidity-dev friendly!

I also made that the page will react automatically when the user changes address or network in the wallet, without needing to the user to refresh the webpage

**Comment by achallagundla** [comment]
Heya @esteblock  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1200 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya
